### DISCUSSION TITLE
Able to disable input icon

### DISCUSSION THREAD

**Discussion by NargiT** [simple-login/app#1887] [Feature Requests]
Hello,

I am using a password manager and I have an issue where for any input (username/email) I see both SmartLogin logo and  my Password Manager icon overlapping.

Since my password manager is used 99% of the time, I would like to disable smart login integration with input. The best case would be to integrate automatically with my password manager...

On account.google.com I have this overlapping
![image](https://github.com/simple-login/app/assets/3153333/0c5d18bc-3fee-4ad1-9448-651f565caafa)
For miro.chI everything is ok
![image](https://github.com/simple-login/app/assets/3153333/914235bf-7709-4a58-aba5-ff48215a1105)

**Comment by nguyenkims** [comment]
@NargiT you can use this option that'll display SL icon outside of the email field

<img width="441" alt="image" src="https://github.com/simple-login/app/assets/3167780/209159b3-6f87-46e3-a1e7-52bcfb3ae0d9">

**Reply by NargiT** [reply]
~~Is this available for firefox extension ? I am not able to find it.~~

Ok I was finally able to find how to access settings. Everything good thanks
